ui/pages/confirmations/components/confirm/info/approve/hooks/use-custom-spending-cap.ts

```diff
@@ -0,0 +1,7 @@
+import { useState } from 'react';
+
+export const useCustomSpendingCap = () => {
```

Comment:
We currently have the context which has it's own state, plus this hook?

Reply:
Looks like this was left to be deleted